whois-update/src/main/java/net/ripe/db/whois/update/domain/Update.java

```diff
@@ -93,11 +94,22 @@ public String toString() {
         return builder.toString();
     }
 
-    public void setEffectiveCredential(final String effectiveCredential) {
+
+    public void setEffectiveCredential(final String effectiveCredential, EffectiveCredentialType effectiveCredentialType) {
```

Comment:
Update should not be mutable, we should find a better way to track the effective credential